spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/source/SparkScan.java

```diff
@@ -147,9 +152,9 @@ protected Statistics estimateStatistics(Snapshot snapshot) {
 
   @Override
   public String description() {
-    String filters =
-        filterExpressions.stream().map(Spark3Util::describe).collect(Collectors.joining(", "));
-    return String.format("%s [filters=%s]", table, filters);
+    return String.format(
+        "%s [filters=%s, groupedBy=%s]",
+        table(), Spark3Util.describe(filterExpressions), groupingKeyType());
```

Comment:
This will be part of Spark UI.

Reply:
Do we want a simpler grouping expression? This is going to show `struct<...>` where we may just want the column names from that struct.

Reply:
Switched to column names only.